website/pages/blog/azure-cost-optimization-with-cloudquery.md

```diff
@@ -0,0 +1,169 @@
+
+## Prerequisites
+
+- [Azure Source Plugin](https://www.cloudquery.io/docs/plugins/sources/azure/overview)
+- [PostgreSQL Destination Plugin](https://www.cloudquery.io/docs/plugins/destinations/postgresql/overview)
+- A running PostgreSQL instance (can be local or remote)
+
+## Create a Cost Analysis Report in Azure
+
+First, we need to create a _Cost Analysis Report_ in Azure. To do that, go to the Cost Management section in the Azure Portal and click on the [Cost Analysis](https://portal.azure.com/#view/Microsoft_Azure_CostManagement/Menu/~/costanalysis/openedBy/AzurePortal) tab.
+
+![Azure Cost Analysis Start Page](/images/blog/azure-cost-optimization-with-cloudquery/cost-analysis-cost-by-resource-report.png)
+
+Then, click on the _Cost by Resource_ button.
+
+![Azure Cost By Resource Menu](/images/blog/azure-cost-optimization-with-cloudquery/cost-analysis-cost-by-resource-menu.png)
+
+Here you can make adjustments to your report. Make sure to select the time range you want to analyze. For this tutorial, we'll leave this at the default of current billing period. Make sure the _Group By_ option is set to **Resource**.
+
+![Azure Cost By Resource Report](/images/blog/azure-cost-optimization-with-cloudquery/cost-analysis-cost-by-resource-report-annotated.png)
+
+Use the _Save as_ button to save this as a new report view. Give it a name and click _Save_.
+
+![Azure Cost By Resource Report Save Button](/images/blog/azure-cost-optimization-with-cloudquery/cost-analysis-cost-by-resource-report-save.png)
+
+From now on every time we run a `cloudquery sync` command, the data from this report will be synced to PostgreSQL database. 
+
+
+## Syncing data
+
+Before going off and writing queries, we need to sync the data from Azure to our PostgreSQL database. To do that, we'll use the `cloudquery sync` command. Let's start with writing (well, copy-pasting) a configuration file (For full config reference, checkout the [Azure Source Plugin](https://www.cloudquery.io/docs/plugins/sources/azure/configuration) and [PostgreSQL Destination Plugin](https://www.cloudquery.io/docs/plugins/destinations/postgresql/overview)):
+
+```yaml copy
+kind: source
+spec:
+  # Source spec section
+  name: "azure"
+  path: "cloudquery/azure"
+  version: "VERSION_SOURCE_AZURE"
+  destinations: ["postgresql"]
+  concurrency: 100 # Azure is a beast: If you're running this on a beefy machine, you can increase this
+  tables: ["*"] # "azure_costmanagement_views" and "azure_costmanagement_view_queries" are required for this tutorial, but let's get them all
+  spec:
+    # We could specify a list of subscriptions to sync, but we'll just sync all of them
+    # subscriptions: ["00000000-0000-0000-0000-000000000000"]
```

Comment:
```suggestion
  tables: ["*"] # "azure_costmanagement_views" and "azure_costmanagement_view_queries" are required for this tutorial, but let's get them all
```

For brevity. Also maybe we can sync just the required table so we can drop the `concurrency` configuration

Reply:
If we just sync the required tables then there's no way of correlating them with other data later, because the rows won't exist.